pages/fundamentals/data-types.mdx

```diff
@@ -664,6 +665,64 @@ ZonedDateTime operations:
 | ZonedDateTime - Duration      | ZonedDateTime |
 | ZonedDateTime - ZonedDateTime | Duration      |
 
+### Enum
+
+Unlike other datatypes, enums requires that they are defined first. Each named enum had a set of values it can represent.
+
+<Callout type="info">
+
+To create an Enum:
```

Comment:
Is there a reason for capital E only in this mention of the word Enum?

Reply:
Consistency with how other some of the other datatypes were referred to. I have not checked if its consistent all the datatypes referenced.